magellan-library/src/main/java/com/wealthfront/magellan/navigation/NavigationDelegate.kt

```diff
@@ -73,32 +72,32 @@ class NavigationDelegate(
     activity = null
   }
 
-  fun goTo(nextNavigableCompat: NavigableCompat) {
-    navigateTo(nextNavigableCompat, GO)
+  fun goTo(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    navigateTo(nextNavigableCompat, overrideTransition ?: DefaultTransition())
   }
 
-  fun show(nextNavigableCompat: NavigableCompat) {
-    navigateTo(nextNavigableCompat, SHOW)
+  fun show(nextNavigableCompat: NavigableCompat, overrideTransition: Transition? = null) {
+    navigateTo(nextNavigableCompat, overrideTransition ?: ShowTransition())
```

Comment:
If we're making this such that `show` is just a transition, we probably should remove `navigator.show()` (while keeping `legacyNavigator.show()` for compatibility)